app-backup/backuppc/backuppc-4.4.0.ebuild

```diff
@@ -0,0 +1,187 @@
+	app-admin/apache-tools
+	app-admin/makepasswd
+	dev-lang/perl
+"
+
+# The CGI modules are handled in ${RDEPEND}.
+APACHE_MODULES="apache2_modules_alias," # RedirectMatch
+APACHE_MODULES+="apache2_modules_authn_core," # AuthType
+APACHE_MODULES+="apache2_modules_authz_core," # Require
+APACHE_MODULES+="apache2_modules_authz_host," # Require host
+APACHE_MODULES+="apache2_modules_authz_user" # Require valid-user
+
+# Older versions of mod_perl think they're compatible with apache-2.4,
+# so we require the new one explicitly.
+RDEPEND="
+	${DEPEND}
+	app-arch/par2cmdline
+	dev-perl/Archive-Zip
+	dev-perl/CGI
+	dev-perl/File-RsyncP
+	dev-perl/libwww-perl
+	dev-perl/BackupPC-XS
+	net-misc/rsync-bpc
+	virtual/mta
+	virtual/perl-IO-Compress
+	www-apache/mod_perl
+	www-apache/mpm_itk
+	|| (
+		>=www-servers/apache-2.4[${APACHE_MODULES},apache2_modules_cgi]
+		>=www-servers/apache-2.4[${APACHE_MODULES},apache2_modules_cgid]
+	)
+	rss? ( dev-perl/XML-RSS )
+	samba? ( net-fs/samba )"
+
+CGIDIR="/usr/lib/backuppc/htdocs"
+CONFDIR="/etc/BackupPC"
+DATADIR="/var/lib/backuppc"
+DOCDIR="/usr/share/doc/${PF}"
+LOGDIR="/var/log/BackupPC"
+need_apache2_4
+
+src_prepare() {
+	default
+
+	# Fix initscript
+	#sed -e 's/runscript/openrc-run/g' -i init.d/src/gentoo-backuppc || die
+
+	# Fix docs location using the marker that we've patched in.
+	sed "s+__DOCDIR__+${DOCDIR}+" -i "lib/BackupPC/CGI/View.pm" || die
+}
+
+src_install() {
+	local myconf
+	myconf=""
+	if use samba ; then
+		myconf="--bin-path smbclient=$(type -p smbclient)"
+		myconf="${myconf} --bin-path nmblookup=$(type -p nmblookup)"
+	fi
+
+	/usr/bin/env perl ./configure.pl \
+		--batch \
+		--bin-path perl=$(type -p perl) \
+		--bin-path tar=$(type -p tar) \
+		--bin-path rsync=$(type -p rsync) \
+		--bin-path ping=$(type -p ping) \
+		--bin-path df=$(type -p df) \
+		--bin-path ssh=$(type -p ssh) \
+		--bin-path sendmail=$(type -p sendmail) \
+		--bin-path hostname=$(type -p hostname) \
+		--bin-path gzip=$(type -p gzip) \
+		--bin-path bzip2=$(type -p bzip2) \
+		--config-dir "${CONFDIR}" \
+		--install-dir /usr \
+		--data-dir "${DATADIR}" \
+		--hostname 127.0.0.1 \
+		--uid-ignore \
+		--dest-dir "${D}" \
+		--html-dir "${CGIDIR}"/image \
+		--html-dir-url /image \
+		--cgi-dir "${CGIDIR}" \
+		--fhs \
+		${myconf} || die "failed the configure.pl script"
+
+	ebegin "Installing documentation"
+
+	pod2man \
+		-errors=none \
+		--section=8 \
+		--center="BackupPC manual" \
+		"${S}"/doc/BackupPC.pod backuppc.8 \
+		|| die "failed to generate man page"
+
+	doman backuppc.8
+
+	# Place the documentation in the correct location
+	dodoc "${D}/usr/share/doc/BackupPC/BackupPC.html"
+	dodoc "${D}/usr/share/doc/BackupPC/BackupPC.pod"
+	rm -rf "${D}/usr/share/doc" || die
+
+	eend 0
+
+	# Setup directories
+	dodir "${CONFDIR}/pc"
+
+	keepdir "${CONFDIR}"
+	keepdir "${CONFDIR}/pc"
+	keepdir "${DATADIR}"/{trash,pool,pc,cpool}
+	keepdir "${LOGDIR}"
+
+	ebegin "Setting up init.d/conf.d/systemd scripts"
```

Comment:
I'm not sure I'd bother with this necessarily but it's not bad either. But no eend? I thuoght (not checked) that we're supposed to match them, but do check.